### DISCUSSION TITLE
Using API

### DISCUSSION THREAD

**Discussion by Shvan-ali** [speech-translator-ext/speech-translator-readme#10] [General]
Hi dear, please is that possible to use your API to build a file extension? If yes, how?

**Comment by kappaflow** [comment]
@Shvan-ali hi, I am not sure that I understand your question. Can you clarify what you are asking for?

The extension is not open sourced, but I can do custom builds for specific needs for a payment.

**Comment by Shvan-ali** [comment]
We need to generate an extension file for ourselves, typically for specific
languages, what do we need just for the dataset to give us? Please advise.

Regards

**Reply by kappaflow** [reply]
Do you need to have a custom extension (as a file) that has the languages predefined?
I can't give you the source code, but I can build the extension for your needs. If you are interested you can contact me by email contact@kappaflow.dev to discuss the details.
